app/api/base_api.rb

```diff
@@ -18,8 +20,11 @@ module BaseAPI
 
     helpers ::Helpers::ErrorHelper
 
-    http_basic do |_username, password|
-      password == Rails.configuration.x.app_password
-    end
+    use Grape::Middleware::OptionalAuth,
+        type: :http_basic,
+        realm: 'API Authorization',
+        proc: ->(_username, password) do
+          password == Rails.configuration.x.app_password
```

Comment:
Do we need to subclass the the Grape auth stuff or can we just update this to return true if the passwords match or passwords are disabled? I don't feel strongly either way but I'm just curious.